### PR TITLE
community-membership.md: add community membership guidelines

### PR DESCRIPTION
**Pull request by estroz**
Add community membership guidelines and membership issue template.

### CODE REVIEW

File: community-membership.md
Comment (camilamacedo86) [inline community-membership.md:14]: May `admins` instead of `subproject owner`?

File: community-membership.md
Comment (camilamacedo86) [inline community-membership.md:41]:
```suggestion
- Have made multiple contributions to the project. A contribution may include, but is not limited to:
```

Comment (camilamacedo86) [reply community-membership.md:41]: Cannot the `OR community` make it questionable?

File: community-membership.md
Comment (camilamacedo86) [inline community-membership.md:42]: Authoring? if the person is not a member so I do not think that is possible to authorize something. 
```suggestion
    - Reviewing PRs on GitHub.
```

File: community-membership.md
Comment (camilamacedo86) [inline community-membership.md:43]:
```suggestion
    - Raising and helping to solve issues on GitHub
```

File: community-membership.md
Comment (camilamacedo86) [inline community-membership.md:69]: IMO just the LGTM's done by reviewers should be counted by the bot.

File: community-membership.md
Comment (camilamacedo86) [inline community-membership.md:119]: what does it mean?

File: community-membership.md
Comment (camilamacedo86) [inline community-membership.md:161]: Can someone from the community become to be a `Subproject Owner/admin`? if not, I think we do not need to add this section.

File: community-membership.md
Comment (jberkus) [reply community-membership.md:14]: the term "subproject owner" is used in a bunch of CNCF projects, so it's fairly well understood

File: community-membership.md
Comment (jberkus) [reply community-membership.md:161]: since Operator Framework is a CNCF project now, yes they can.  If they can't, OF's advancement in the CNCF will be blocked.

So if there's more things we need to do in order to make it possible to have community subproject owners, we need to address them.

File: community-membership.md
Comment (jberkus) [reply community-membership.md:43]: +1, since random comments aren't valuable.

File: community-membership.md
Comment (jberkus) [reply community-membership.md:42]: Anyone can submit a PR.  They can't merge it, but they can certainly write it.  -1

File: community-membership.md
Comment (jberkus) [reply community-membership.md:41]: I think the emphasis is meant to indicate that contributions outside of Github can count -- organizing a set of OF workshops, for example.

File: community-membership.md
Comment (jberkus) [inline community-membership.md:20]: do we have a "getting started for contributors" doc?  If so, we should link it here (if not, lemme add a TODO).

**Comment by estroz** [comment]
I've been busy with the SDK v1.0 release, will get around to these comments soon.

File: community-membership.md
Comment (jberkhahn) [inline community-membership.md:111]: Isn't this redundant with "Tests can be run against their PRs automatically. No `/ok-to-test` needed." from being a member?

File: community-membership.md
Comment (jberkhahn) [reply community-membership.md:119]: I think this means hidden repos for keeping track of hidden state like keys and stuff for test infrastructure and junk? Not sure, but I would like this clarified.

File: community-membership.md
Comment (jberkhahn) [reply community-membership.md:69]: Yeah, assuming we're just trying to mirror the way Kube has stuff set up, shouldn't this be a thing for Reviewers?

**Comment by jberkhahn** [comment]
I think we need a section for community co-chairs. There might(?) have been a plan to add that in a separate PR but I can't find any reference to that, it might have gotten lost in the shuffle.

File: community-membership.md
Comment (estroz) [reply community-membership.md:20]: No, will add a TODO.

File: community-membership.md
Comment (estroz) [reply community-membership.md:69]: The goal of letting members `/lgtm` is to allow them to build up to reviewer status by actually getting review experience, therefore I want this requirement to stick around. I don't think tide will merge a PR unless a reviewer assigned to a PR gives a `/lgtm`.

File: community-membership.md
Comment (estroz) [reply community-membership.md:119]: This is Github read access, which is an organization setting.

**Comment by estroz** [comment]
@jberkhahn you mean community co-chair election? There should be a (s)election process for appointment, so that should be tackled in a separate document. I don't have any plans to work on one personally.

**Comment by jberkhahn** [comment]
Yeah, I mean a section that says what a community co-chair is, what they do. I thought we had discussed that in the past but can't find that anywhere.

File: community-membership.md
Comment (jberkhahn) [reply community-membership.md:70]: I'm not entirely clear on how the current OWNERS are going to be fit into these new roles. Does that mean a person will have to be a Reviewer for their LGTM to actually count for merge purposes, or an Approver? or what?

Comment (estroz) [reply community-membership.md:70]: AFAIK your review only counts if you are assigned to a PR.

File: community-membership.md
Comment (asmacdo) [inline community-membership.md:112]: "may be" is a little unclear. I assume that this precondition would be made by existing reviewers/approvers as part of a PR review?

File: community-membership.md
Comment (asmacdo) [reply community-membership.md:20]: Here's what we currently have. This doc should be mentioned somewhere in these docs
https://github.com/operator-framework/operator-sdk/blob/master/CONTRIBUTING.MD
https://sdk.operatorframework.io/docs/contribution-guidelines/

**Review by asmacdo — APPROVED** [review]

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

File: community-membership.md
Comment (estroz) [reply community-membership.md:112]: This would read better as: "Large code contributions by non-reviewers is acceptable at the discretion of existing PR reviewers"

File: community-membership.md
Comment (jberkhahn) [reply community-membership.md:70]: OK. Could we write that down somewhere? I think it's a good idea for the on-ramp for new contributors to be as simple and well-documented as possible. Now that I think about it, a New Contributers's Guide section to the How To Contribute page in the regular docs might be a good place to put this sort of stuff...

Comment (estroz) [reply community-membership.md:70]: Wrote down this label requirement. Still need a new contributor's guide though.

**Pull request merged**
